CONTRIBUTING.md

```diff
@@ -63,6 +63,17 @@ Try those commands below:
 2. Run unit test: `make unit-test`
 3. All of the above: `make`
 
+### Tips
+Using default Python Package Index (PyPI, https://pypi.org/) maybe pretty slow due to the network connection.
+Developers can choose using a PyPI mirror, search engine can help you find nearest and fast PyPI mirror.
+
+Setting python using a PyPI mirror as default package source is easy. For pip version >=10.0.0 (if not satisfied, please using `pip install -U pip` to upgrade pip):
+```bash
+pip config set global.index-url https://pypi.tuna.tsinghua.edu.cn/simple
```

Comment:
So, I have a small issue including this command in the Tips section for all users. Realistically `configure.sh` is only run once per build and I don't really think the speed of the pypi server is the issue here.  I worry that people will just lazily read through this guide and set their global pypi repo to the university simpy because it was a tip.

I would be okay with merging this if we simplified this to a single line saying something like:

**Tip**: If having trouble with running this script due to network connection, users may find want to [consult this documentation](https://mirror.tuna.tsinghua.edu.cn/help/pypi/) to properly set their pypi repository.

Reply:
That's a good idea, more concise.

Reply:
Note the mirror site is designed for programmers in China, how about?

Tip: If having trouble with running this script due to network connection in China, users ...

Reply:
Sounds good to me! I already update the doc according to this.